tests/core/cycle/test_cycle_manager.py

```diff
@@ -241,15 +241,28 @@ def test_get_primary(tmpdir, cycle, current_datetime):
     cycle_2 = _CycleManager._get(cycle_2.id)
     cycles = _CycleManager._get_all()
     assert len(_CycleManager._get_all()) == 2
-    assert len(_CycleManager._get_cycles_by_frequency_and_start_date(cycle_1.frequency, cycle_1.start_date, cycles)) == 1
-    assert len(_CycleManager._get_cycles_by_frequency_and_start_date(cycle_2.frequency, cycle_2.start_date, cycles)) == 1
     assert (
-        len(_CycleManager._get_cycles_by_frequency_and_start_date(Frequency.WEEKLY, datetime(2000, 1, 1, 1, 0, 0, 0), cycles))
+        len(_CycleManager._get_cycles_by_frequency_and_start_date(cycle_1.frequency, cycle_1.start_date, cycles)) == 1
+    )
+    assert (
+        len(_CycleManager._get_cycles_by_frequency_and_start_date(cycle_2.frequency, cycle_2.start_date, cycles)) == 1
+    )
+    assert (
+        len(
+            _CycleManager._get_cycles_by_frequency_and_start_date(
+                Frequency.WEEKLY, datetime(2000, 1, 1, 1, 0, 0, 0), cycles
+            )
+        )
```

Comment:
The formatting looks strange, no?

Reply:
I'll run black again to make sure.

Reply:
That's the way that black is formatting this line, because the name of the methods are too long, its not able to fit everything into one or even two lines.